src/test/resources/arquillian-tests/skipWithXmlManaged/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
     }
 }
 
-project.buildDir = '../build'
+project.getLayout().setBuildDirectory('../build');
```

Comment:
should this really have the `;` at the end?

Reply:
when using groovy, semicolon is optional. it works with or without semicolon